server/lobbyconnection.py

```diff
@@ -427,8 +428,9 @@ def on_connection_made(self, protocol: QDataStreamProtocol, peername: Address):
 
         self.user_agent = message.get('user_agent')
         version = message.get('version')
-        server.stats.gauge('user.agents.None', -1, delta=True)
-        server.stats.gauge('user.agents.{}'.format(self.user_agent), 1, delta=True)
+        metrics.user_connections.labels("None").dec()
```

Comment:
BTW I'm not sure how correctly this code works. Right now these numbers are always wrong, especially because the gauges don't seem to reset to 0 correctly when the server is stopped. So if you can think of a better way to track user agents, then go for it.

Reply:
Previously, they seem to have been set to zero on server start manually here:
https://github.com/FAForever/server/blob/afacbce94e4303d1f6c2e148652985c4769aafdb/server/servercontext.py#L28-L33

I assume that was meant to counteract them not being set to zero on server shutdown? However, it seems weird that they don't start out at zero in a new python process, anyways.
We should check again after testing `prometheus_client` on the test sever.

Reply:
Looking at this again, it seems that `user_agents.None` is decreased, every time the client sends the `ask_session` command, and I assume we cannot guarantee that this is only sent once and I can reproduce this locally.

Then I know what to do :)

Reply:
I just recently added the setting to 0 stuff, but it didn't seem to work at all when we deployed it. I didn't really test it, I just threw it in there hoping it would solve the problem.